pasta_pl.txt

```diff
@@ -80,4 +80,24 @@ Albo Die With Me
 Albo Blipa
 Albo Yo
 Albo Larka
+Albo Jami
+Albo Status.im'a
+Albo Scuttlebutta
+Albo Semaphora
+Albo stealthy.im'a
+Albo Fortknoxtera
+Albo Crypvisera
+Albo Tungstena
+Albo ȱ
+Albo vero
+Albo tinode'a
+Albo Dust'a
+Albo ZeroMQ
```

Comment:
Does this count as communicator? Afair it's just a library/protocol

Reply:
I'm a simple dude, I see something at [digital communications protocol google sheet](https://docs.google.com/spreadsheets/d/1-UlA4-tslROBDS9IqHalWVztqZo7uxlCeKPQ-8uoFOU/edit), I copy it here.

Reply:
And I'm not alone in this, major part ought this list has entries from that sheet in the very same order in some parts

Reply:
this conversation is opressing, and break CoC that will never exist here. Pull request accepted with dusgust.